### PR TITLE
TASK-37051: Refix redirect URL after clicking on the space logo

### PR DESCRIPTION
**Pull request by GouadriaHanen**
spaceGroupUri is not updated when selecting a space always redirect to the same space due to spaceContact is not updated
spaceContact must be updated

### CODE REVIEW

File: application/src/main/webapp/vue-app/components/ExoChatContact.vue [hunk 1]
```diff
@@ -163,6 +163,7 @@ export default {
       return this.type === 'u' && !this.isEnabled ? 'inactive' : 'active';
     },
     spaceGroupUri() {
+      this.getSpace();
```
Comment (ahamdi) [inline application/src/main/webapp/vue-app/components/ExoChatContact.vue]: If this function is not reused somexhere else, just put the instructions here and remove it.

File: application/src/main/webapp/vue-app/components/ExoChatContact.vue [hunk 2]
```diff
@@ -118,7 +118,7 @@ export default {
         invisible: this.$t('exoplatform.chat.invisible'),
         offline: this.$t('exoplatform.chat.button.offline')
       },
-      spaceContact: {},
+      spaceGroupUri : '',
```
Comment (ahamdi) [inline application/src/main/webapp/vue-app/components/ExoChatContact.vue:121]: Do we need this data property ?

Comment (GouadriaHanen) [reply application/src/main/webapp/vue-app/components/ExoChatContact.vue:121]: yes to avoid infinity loop and promise

**Review by ahamdi — APPROVED** [review]

**Comment by hbenali** [comment]
#234 Complementary fix

**Pull request merged**